src/ops/inc_multihead_self_attention.cu

```diff
@@ -542,91 +542,32 @@ template <typename DT>
 void compute_qkv_kernel(IncMultiHeadSelfAttentionMeta const *m,
                         BatchConfig const *bc,
                         int shard_id,
-                        DT const *input_ptr,
-                        DT const *weight_ptr,
+                        // DT const *weight_ptr,
```

Comment:
can you go ahead and delete the weight_ptr and bias_ptr from any function that still has still as a parameter, even if commented out? You might need to also update the headers